Modules/EMCAL/src/PedestalChannelCheck.cxx

```diff
@@ -0,0 +1,202 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   PedestalChannelCheck.cxx
+/// \author Sierra Cantway
+///
+
+#include "EMCAL/PedestalChannelCheck.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/Quality.h"
+#include "QualityControl/QcInfoLogger.h"
+#include <fairlogger/Logger.h>
+#include <DataFormatsQualityControl/FlagType.h>
+#include <DataFormatsQualityControl/FlagTypeFactory.h>
+// ROOT
+#include <TH1.h>
+#include <TH2.h>
+#include <TPaveText.h>
+#include <TLatex.h>
+#include <TList.h>
+#include <TRobustEstimator.h>
+#include <TMath.h>
+#include <ROOT/TSeq.hxx>
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+namespace o2::quality_control_modules::emcal
+{
+
+void PedestalChannelCheck::configure()
+{
+  // configure threshold-based checkers for bad quality
+  auto nBadPedestalChannelADCRangeLow = mCustomParameters.find("BadPedestalChannelADCRangeLow");
+  if (nBadPedestalChannelADCRangeLow != mCustomParameters.end()) {
+    try {
+      mBadPedestalChannelADCRangeLow = std::stof(nBadPedestalChannelADCRangeLow->second);
+    } catch (std::exception& e) {
+      ILOG(Error, Support) << fmt::format("Value {} not a float", nBadPedestalChannelADCRangeLow->second.data()) << ENDM;
```

Comment:
fmt library not needed, please change the next iteration. As example, the line should look like
```c++
ILOG(Error, Support) << "Value " << nBadPedestalChannelADCRangeLow->second << " not a float" << ENDM;
```
Also, in case of C++ streams, for all objects that implement operator<< (among them std::string) you can put the object directly on the stream, as shown above. There is no need to access the C-string from the std::string. This is only necessary if you use print- or logging functions.